docs/code/modules.rst

```diff
@@ -82,6 +82,12 @@ Encoders
 .. autoclass:: texar.modules.RNNEncoderBase
     :members:
 
+:hidden:`XLNetEncoder`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. autoclass:: texar.modules.XLNetEncoder
+    :members:
+    :exclude-members: _execute
```

Comment:
Do not expose `_execute` to users, only `_build`

Reply:
I think exclude members means to not show the function doc